dotcom-rendering/src/components/marketing/epics/ctas/EpicButton.tsx

```diff
@@ -91,13 +109,22 @@ export const EpicButton: ReactComponent<Props> = (allProps: Props) => {
 			);
 		}
 	};
+	const isColourInTestVariant =
```

Comment:
this is already determined elsewhere - could we not pass `isColourInTestVariant` into this component instead?

Reply:
yes, we can , but all the required details are already available in tracking for the ContributionsEpicCtasContainer, ContributionsEpicButtons, PrimaryCtaButton, so  thought would be better to redefine isColourInTestVariant from tracking at EpicButton